src/rtext.c

```diff
@@ -1371,15 +1371,25 @@ const char *TextFormat(const char *text, ...)
 
     va_list args;
     va_start(args, text);
-    vsnprintf(currentBuffer, MAX_TEXT_BUFFER_LENGTH, text, args);
+    int charCountRequired = vsnprintf(currentBuffer, MAX_TEXT_BUFFER_LENGTH, text, args);
     va_end(args);
 
+    // If charCountRequired is larger than the MAX_TEXT_BUFFER_LENGTH, then overflow occured
+    if(charCountRequired > MAX_TEXT_BUFFER_LENGTH)
+    {
+        // We are going to insert [TRUN] at the end of the string so the user knows what happened
+        char *truncBuffer = buffers[index] + MAX_TEXT_BUFFER_LENGTH - 7; // 7 = six letters + '\0'
+        sprintf(truncBuffer, "[TRUN]");
+        TRACELOG(LOG_WARNING, "RTEXT: TextFormat string was [TRUN]cated. If you need longer strings, please increase MAX_TEXT_BUFFER_LENGTH.");
```

Comment:
@Murlocohol Probably should remove this `TRACELOG` as per https://github.com/raysan5/raylib/issues/3366#issuecomment-1743836669:
> I think in this specific case it could log some warning... but for this specific function the warning will be logged 60 times per second, that's something that I also avoid in raylib.

Reply:
👍 Removed